backend/environment/env.graph-service.template

```diff
@@ -0,0 +1,85 @@
+# Copy this file to "<project_root>/.env.graph-service", and then tweak values for local development
+# Values in this file will override the same-named environnent variables in `.env.common.docker` for the graph-service
+
+# Max number of jobs allowed on the 'graphUpdateQueue' before
+# blockchain scan will be paused to allow queue to drain
+QUEUE_HIGH_WATER=1000
+
+# Number of seconds to retain pending graph updates in the Redis cache to avoid redundant fetches from the chain
+DEBOUNCE_SECONDS=10
+
+# Maximum amount of provider capacity this app is allowed to use (per epoch)
+#     type: 'percentage' | 'amount'
+#     value: number (may be percentage, ie '80', or absolute amount of capacity)
+CAPACITY_LIMIT='{"type":"percentage", "value":80}'
+
+# Graph environment type. This can be 'Dev' or 'Rococo', 'TestnetPaseo', or 'Mainnet'.
+GRAPH_ENVIRONMENT_TYPE=Dev
+
+# [NOTE]: The following config is only used for Dev environments.
+# Add the graph environment config in JSON format only used for Dev environments.
+# Be careful to escape any inner quotes as this is in a .env file.
+GRAPH_ENVIRONMENT_DEV_CONFIG='{
```

Comment:
This eventually gets replaced with the schema names, but I guess we need it for now

Reply:
Right.